src/ml/inc/brainflow_model_params.h

```diff
@@ -21,7 +21,7 @@ struct BrainFlowModelParams
     }
 
     // default copy constructor and assignment operator are ok, need less operator to use in map
-    bool operator< (const struct BrainFlowModelParams &other) const
```

Comment:
here too

Reply:
@Andrey1994 i think i changed these on accdient while changing the other functions arguments.

Reply:
They should still be const and pass by value.

Reply:
Could you restore it here and in other places?

Reply:
ok i will change them back